lang/src/main/antlr4/org/kaazing/k3po/lang/parser/v2/Robot.g4

```diff
@@ -57,10 +61,10 @@ optionNode
     ;
 
 writeOptionNode: 
-    k=WriteKeyword OptionKeyword name=MaskKeyword value=writeValue;
+    k=WriteKeyword OptionKeyword (name=MaskKeyword value=writeValue | name=OffsetKeyword DecimalLiteral);
```

Comment:
I would be putting offset=DecimalLiteral so that it won't clash with "writeValue". But that's only for processing it doesn't affect the syntax.